crates/astria-composer/tests/blackbox/helper/mock_sequencer.rs

```diff
@@ -0,0 +1,41 @@
+//! A minimal mocked sequencer that implements the `broadcast_tx_sync` JSONRPC.
```

Comment:
I have removed that doc comment because it's confusing. It's just for starting a mock server and mounting the `abci_info` response by default. The `broadcast_tx_sync` mock is mounted in the test.

Since this is using wiremock without providing new abstractions the doc comment was more confusing than useful.